### DISCUSSION TITLE
I can't get push notifications to come in

### DISCUSSION THREAD

**Discussion by JacksonCampbell99** [community/community#5816] [Mobile]
I have all my notifications on in the app and phone settings for the app, but all I get is an email and no push notifications for all my GitHub activity. It could be a beta bug. Whatever it is, I'd appreciate if they did work or it was fixed.

**Comment by JacksonCampbell99** [comment]
Anybody going to respond?

**Reply by mxie** [reply]
Hi @JacksonCampbell99 - are you on iOS or Android? Are you still not receiving any notifications?

**Reply by JacksonCampbell99** [reply]
Android. With this latest update today I am receiving some. It seems to only be direct messages and not all of them. I'm not sure if I'm supposed, but I'm not getting notifications on my issues when people comment, only emails.

**Reply by mxie** [reply]
The `Notifications` tab in the app should list the same notifications as you'd see in the web experience or in your emails. Push notifications will only be triggered for whatever you enabled in the app, e.g. direct mentions, assignments, and various review requests, and these might not necessarily match what you see under `Notifications`.

Is this helpful? Is there any way that you can reproduce the issue reliably for us?

**Reply by JacksonCampbell99** [reply]
None of that is helpful since I'm a technical specialist and I have all the notifications settings in the app on and all the notification settings for the app in Android settings on.

**Reply by JacksonCampbell99** [reply]
I will continue to watch what notifications I get and don't get and report to me which aren't coming in.

**Reply by mxie** [reply]
Sounds good. Thanks! If you can identify any patterns in what you're receiving or not receiving, we'll be in a better position to help you out.

**Reply by JacksonCampbell99** [reply]
Well, I didn't receive your response just now, so that's an example.

**Reply by mxie** [reply]
Ah, I see. Discussions and discussion comments are not supported at this time. As I mentioned earlier, only direct mentions, assignments, and review requests are available for push notifications. You should, however, have seen a notification in your email or app notification inboxes though, which are managed differently.

**Reply by JacksonCampbell99** [reply]
I'll keep watching for the others. I guess I should submit feedback for all things in notifications to have an option to notify on the phone.

**Reply by TC72** [reply]
I'm not receiving any notifications from the App on my phone. I was mentioned directly as part of an issue I was working on and didn't receive anything on the App.

Here's the thread, you can see @TC72 from the start of December. This is happening all over this project I'm trying to help out with and gives the impression I've just vanished.

https://github.com/nvim-telescope/telescope.nvim/pull/1401#issuecomment-994091683

Here are my settings inside the github App:
![IMG_1636](https://user-images.githubusercontent.com/402357/148758492-fb4efd08-02b6-400b-8550-120e68017f22.png)

**Reply by pwmcintyre** [reply]
i'm experiencing the same issue
- android; have tried to un/re-install
- enabled all notificiations via app / confirmed via web
- confirmed; i get lots of notifications appear in my [web notifications](https://github.com/notifications) page; and the notifications tab on mobile app
- and yet; no push notifications

edit: ok i appear to be getting push notifications specifically for "review comments" (but not comments)
